tests/src/Functional/NodeFormMultipleTest.php

```diff
@@ -162,6 +179,19 @@ private function runFieldTest($field_type = 'options_select') {
         $field_name . '[' . $staff_term->id() . ']' => NULL,
       ];
     }
+    if ($field_type === 'entity_reference_autocomplete') {
+      $page->findField($field_name . '[0][target_id]');
+      $page->findField($field_name . '[1][target_id]');
+      $web_assert->fieldNotExists($field_name . '[2][target_id]');
+      $edit = [
+        $field_name . '[0][target_id]' => $base_term->getName() . ' (' . $base_term->id() . ')',
+        $field_name . '[1][target_id]' => NULL,
+      ];
+      // This all works to this point, but for autocomplete handling, the
```

Comment:
@larowlan So all of the handling of autocomplete works in browser but not in this test.

The 'value' field for the disallowed options is not being processed, though it works fine for selects and checkboxes.

Any ideas?

Reply:
no sorry, might need to fire up xdebug there to see the difference between the posted values in both cases.

Reply:
It was a problem with altering the form in two places.

It might be cleaner if we taught disallowedOptions() to understand entity_autocomplete, as right now we are altering form data in two separate places.

Reply:
It doesn't look like disallowedOptions() can do that without changing the method signature, so I think we're stuck with Taxonomy autocompletes being a special case.